### PR TITLE
Makeshift gun renames

### PR DESCRIPTION
**Pull request by Humonitarian**
## About The Pull Request

All names from the Foundation series by Isaac Asimov.

## Why It's Good For The Game

New names cool,

## Changelog
:cl:
tweak: Many HomeMadeTM guns renamed
/:cl:

### CODE REVIEW

File: code/modules/projectiles/guns/projectile/battle_rifle/boltgun.dm [hunk 1]
```diff
@@ -176,7 +176,6 @@
 	w_class = ITEM_SIZE_HUGE
 	slot_flags = SLOT_BACK
 	damage_multiplier = 1.2
-	penetration_multiplier = 1.3
```
Comment (zombiemimic) [inline code/modules/projectiles/guns/projectile/battle_rifle/boltgun.dm:179]: is this an intended removal?

Comment (MLGTASTICa) [reply code/modules/projectiles/guns/projectile/battle_rifle/boltgun.dm:179]: LE EPIC STEALTH NERF

Comment (SirRichardFrancis) [reply code/modules/projectiles/guns/projectile/battle_rifle/boltgun.dm:179]: Balance PR 7116: Sneaky Edition

Comment (Humonitarian) [reply code/modules/projectiles/guns/projectile/battle_rifle/boltgun.dm:179]: how did that get there

Comment (vode-code) [reply code/modules/projectiles/guns/projectile/battle_rifle/boltgun.dm:179]: Congrats, The Curse has been transferred to you.

Comment (Humonitarian) [reply code/modules/projectiles/guns/projectile/battle_rifle/boltgun.dm]: Should be fixed

**Review by zombiemimic — APPROVED** [review]
With the fix, everything looks good

**Review by Firefox13 — APPROVED** [review]

**Pull request merged**